src/Compilers/CSharp/Test/Semantic/Semantics/PatternMatchingTests.cs

```diff
@@ -4914,7 +4914,7 @@ public class Program
                 );
         }
 
-        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/16696")]
+        [Fact, WorkItem(16696, "https://github.com/dotnet/roslyn/issues/16696")]
         public void TypeParameterSubsumption04()
```

Comment:
>TypeParameterSubsumption04 [](start = 20, length = 26)

Please adjust the test scenario so that it could be executed to verify that all cases are reachable at runtime.